Sources/swift-parsing-benchmark/JSON.swift

```diff
@@ -48,24 +48,26 @@ private let object = Skip(StartsWith("{".utf8))
         .take(stringLiteral)
         .skip(Whitespace())
         .skip(StartsWith(":".utf8))
-        .skip(Whitespace())
```

Comment:
Removed some excess `Whitespace()` parsers (the parser that comes after is surrounded by them).